pkg/rbac/clusterScopedResources_test.go

```diff
@@ -0,0 +1,112 @@
+package rbac
+
+import (
+	"testing"
+	"time"
+
+	"github.com/driftprogramming/pgxpoolmock"
+	"github.com/golang/mock/gomock"
+)
+
+// Initialize cache object to use tests.
+func newResourcesListCache(t *testing.T) (*pgxpoolmock.MockPgxPool, Cache) {
```

Comment:
- The `MockPgxPool` is included in the `Cache` so you only need to return the Cache here.
- I'd call it `mockResourcesListCache()` for clarity.
```suggestion
// Initialize a mock cache object to use in tests.
func mockResourcesListCache(t *testing.T) (Cache) {
```

Reply:
This is related issue with replacing pgxpool too pgxpoolmock in connection.go.

Reply:
If I remove the returned pool, the cache passes pgxpoolmock instead of *pgxpoolmock (since in Cache.go we have pgxpoolmock)

Reply:
Ahh, ok got it.  Thanks for clarifying it!